app/femr/business/services/system/UserService.java

```diff
@@ -299,6 +298,11 @@ private void encryptAndSetUserPassword(IUser user) {
         user.setPassword(encryptedPassword);
     }
 
+    public Boolean checkOldPassword(String newpassword, String oldpassword)
```

Comment:
This needs the @override annotation to tell the compiler you are overriding a method in the IUserService interface (see other methods in the same class)